packages/react/src/components/Breadcrumb/BreadcrumbLink.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { forwardRef } from 'react';
 import classnames from 'classnames';
-import { Cauldron } from '../../types';
 
-interface BreadcrumbLinkProps extends React.HTMLAttributes<HTMLLinkElement> {
+interface BreadcrumbLinkProps
+  extends Omit<React.LinkHTMLAttributes<HTMLLinkElement>, 'as'> {
```

Comment:
Is the `Omit<..., 'as'>` necessary? Was there a conflict?

Reply:
I ended up doing this, because `as` is typed as a `string` from `HTMLAttributes` vs `React.ElementType` in `BreadcrumbLinkProps`